backend/routes/orders.js

```diff
@@ -1,16 +1,61 @@
 const { authSellerAndShop } = require('./_auth')
-const { Order } = require('../models')
+const {
+  Order,
+  sequelize,
+  Sequelize: { Op }
+} = require('../models')
 const { findOrder } = require('../utils/orders')
 const makeOffer = require('./_makeOffer')
 const { sendNewOrderEmail } = require('../utils/emailer')
 
 module.exports = function (app) {
   app.get('/orders', authSellerAndShop, async (req, res) => {
-    const orders = await Order.findAll({
-      where: { shopId: req.shop.id },
-      order: [['createdAt', 'desc']]
+    const { page: pageVal, search } = req.query
+
+    const page = parseInt(pageVal) || 1
+    const limit = parseInt(process.env.ORDER_PAGINATION_LIMIT) || 50
+    const offset = page ? (page - 1) * limit : undefined
+
+    const where = {
+      shopId: req.shop.id
+    }
+
+    if (search) {
+      where[Op.or] = [
+        sequelize.where(
+          sequelize.cast(sequelize.col('data'), 'text'),
+          'ILIKE',
```

Comment:
Is ILIKE supported by sqlite? I think it's Postgres only.
For local development and unit testing it would probably be nice to continue supporting sqllite.
Perhaps add a check on the sequelize dialect and fallback to LIKE if sqlite?